spec/util/http.spec.ts

```diff
@@ -168,19 +170,23 @@ describe('uri', function (): void {
 	describe('#getOpUrl()', function (): void {
 		it('returns a URL', function (): void {
 			expect(
-				getOpUrl('https://localhost:8080', '', 'api', 'demo', 'ops')
+				getOpUrl(DEFAULT_OPS_URL_TEMPLATE, {
```

Comment:
Having a `defaultOpUrl` function that composes the template and prefix, would probably be easier to use.

Reply:
Yes that's a great idea. I'll rework this with optional functions that can be passed in.